src/QMCDrivers/VMC/VMCDriverInput.cpp

```diff
@@ -0,0 +1,35 @@
+//////////////////////////////////////////////////////////////////////////////////////
+// This file is distributed under the University of Illinois/NCSA Open Source License.
+// See LICENSE file in top directory for details.
+//
+// Copyright (c) 2019 QMCPACK developers.
+//
+// File developed by: Peter Doak, doakpw@ornl.gov, Oak Ridge National Laboratory
+//
+// File created by: Peter Doak, doakpw@ornl.gov, Oak Ridge National Laboratory
+//////////////////////////////////////////////////////////////////////////////////////
+
+#include "QMCDrivers/VMC/VMCDriverInput.h"
+
+namespace qmcplusplus
+{
+VMCDriverInput::VMCDriverInput(int walkers_per_rank, const std::string& use_drift)
```

Comment:
Why do we need walkers_per_rank and walkers_per_rank? I think read from XML is sufficient. I found them used for testing. If testing is the only purpose, this needs to be changed.

Reply:
walkers_per_rank and walkers_per_crowd?
Ideally you should be able to construct these input objects without XML although at this point the Constructors are not very systematic.